simiyu1/JavaDevsNairobi
# 155965773 create githubusers adapter

#### What does this PR do?
 - Create github users adapter
#### Description of Task to be completed?
 - create package adapter
 - create class GithubUsersAdapter
 
#### How should this be manually tested?
 - On navigation to the adapter directory and you should see the [ GithubUsersAdapter](https://github.com/simiyu1/JavaDevsNairobi/blob/ft-create-githubusers-adapter-155965773/app/src/main/java/com/example/simiyunabende/javadevs/adapter/GithubUsersAdapter.java) file

#### Any background context you want to provide?
 - The presenter class is in-charge of updating data into the view, it uses the adapter class to do this. 

#### What are the relevant pivotal tracker stories?
 - [#155965773](https://www.pivotaltracker.com/story/show/155965773)
